jsdelivr-npm-importmap.js

```diff
@@ -38,12 +38,16 @@ const importMap = {
       "https://cdn.jsdelivr.net/npm/@marlowe.io/language-core-v1@0.4.0-beta-rc1/dist/bundled/esm/version.js",
     "@marlowe.io/language-examples":
       "https://cdn.jsdelivr.net/npm/@marlowe.io/language-examples@0.4.0-beta-rc1/dist/bundled/esm/language-examples.js",
+    "@marlowe.io/language-examples/atomic-swap":
```

Comment:
@nhenin why is this a top level import? shouldn't it be accessible by the language-examples directly?

Reply:
fixed